server/integration/testsuite/src/test/java/org/infinispan/server/test/client/rest/AbstractRESTClientIT.java

```diff
@@ -55,9 +55,13 @@ public String getContent() {
         }
     }
 
+    protected String getDefaultCache() {
+        return "default";
+    }
+
     @Before
     public void setUp() throws Exception {
-        rest = new RESTHelper();
```

Comment:
It seems this call has exactly the same result as previously. So perhaps we could either revert it to `rest = new RESTHelper()` or remove default constructor from `RESTHelper`?

Reply:
The default ctor is used in several other places, and the effect of this change is to allow inheritors of ```AbstractRESTClientIT``` to override the cache name used in the test,